event:
  - id: 10
    name: 'E-STOP'
    animation:
      both:
        image: test_pattern.png
        # color: 0xFFFF00
        duration: 0.1
        keep_state: True

  - id: 25
    name: 'TURN-LEFT'
    animation:
      front:
        image: blinker.png
        duration: 2
        repeat: 2
        keep_state: True

      tail:
        image: blinker.png
        duration: 3
        keep_state: False


  - id: 21
    name: 'TURN-LEFAT'
    interrupting: True
    animation:
      both:
        animation_name: battery
        duration: 2


  - id: 2137
    name: 'SLIDE-FIVE'
    animation:
      both:
        image: blinker.png
        duration: 1
        repeat: 2
        keep_state: True
